riposte-spi/src/main/java/com/nike/riposte/server/http/Endpoint.java

```diff
@@ -117,4 +119,13 @@ default ObjectMapper customResponseContentSerializer(@SuppressWarnings("UnusedPa
         return null;
     }
 
+    /**
+     * @return The max request size in bytes that you want to allow for this specific endpoint, or Optional.empty()
+     * if you want to use the app-wide default max request size
+     * returned by {@link ServerConfig#maxRequestSizeInBytes()} ()}.
```

Comment:
Trailing `()}` that needs to be deleted. Also, the javadoc should include something about returning 0 to disable request size validation entirely.